composer.json

```diff
@@ -21,39 +16,17 @@
     "extra": {
         "branch-alias": {
             "dev-master": "0.11.x-dev"
+        },
+        "acorn": {
+            "providers": "Itineris\\SageFLBuilder\\SageFLBuilderServiceProvider"
```

Comment:
Does this need to be mentioned in the README? I do not know.

Reply:
@codepuncher, I think not.